modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -1221,109 +1221,6 @@
 		['aui-base', 'liferay-util-window']
 	);
 
-	/**
-	 * @deprecated As of Cavanaugh (7.4.x), replaced by `openSelectionModal`
-	 */
-	Liferay.provide(
-		Util,
-		'selectEntity',
```

Comment:
We are using this util in 21 placed in our monorepo, in `npm-scripts`, for example [here](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/npm-tools/packages/npm-scripts/__fixtures__/jsp/roles.jsp#L152). @bryceosterhaus Any idea what this is for? Can we ignore it?

Reply:
I can remove the fixtures from npm-scripts too :scream: :scream: :scream:

Reply:
or ignore them...

Reply:
I think the fixtures in npm-scripts is for testing our "prettier" command. So yeah they can be removed in npm-scripts or ignore them since they aren't used for anything other than just testing formatting.